src/components/BookForm.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/booksSlice';
+
+const BookForm = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: Date.now().toString(),
+      title,
+      author,
+    };
+    dispatch(addBook(newBook));
+    setTitle('');
+    setAuthor('');
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <h2>ADD NEW BOOK</h2>
+      <input
+        type="text"
+        placeholder="Title"
+        value={title}
+        onChange={(e) => setTitle(e.target.value)}
+      />
+      <input
+        type="text"
+        placeholder="Author"
+        value={author}
+        onChange={(e) => setAuthor(e.target.value)}
+      />
+      <button type="submit">ADD BOOK</button>
```

Comment:
- [ ] Hi @noelfoka, the event handler(onClick) to add a book to the book list should be implemented on this `<button></button>` as you can see on the screenshot below not on the form itself. Let's kindly fix :wrench: it

![Screenshot from 2023-08-23 05-20-18](https://github.com/noelfoka/bookstore/assets/63087491/7e96e7d3-e788-4c09-bb3e-2a39caa1d636)

Reply:
> - [ ] As requested by the code reviewer, kindly add the event handler(onClick) to add a book to the book list on the button, not the form itself.👍